build/patch-charts.sh

```diff
@@ -38,42 +38,47 @@ function patchTemplates() {
 
   # - remove istio-multi service account
   # - for 1.6, this contains the sa for istiod, so need to do some munging
-  sed_wrap -i -e 's/\(: istiod-service-account\)/\1-{{ .Values.revision | default "default" }}/' \
-      ${HELM_DIR}/base/templates/serviceaccount.yaml \
-      ${HELM_DIR}/base/templates/rolebinding.yaml \
-      ${HELM_DIR}/base/templates/clusterrolebinding.yaml \
-      ${HELM_DIR}/istio-control/istio-discovery/templates/deployment.yaml
-
-  sed_wrap -i -e '1,/^---/ d' \
-    ${HELM_DIR}/base/templates/serviceaccount.yaml \
-    ${HELM_DIR}/base/templates/clusterrolebinding.yaml
+#  sed_wrap -i -e 's/\(: istiod-service-account\)/\1-{{ .Values.revision | default "default" }}/' \
+#      ${HELM_DIR}/base/templates/clusterrolebinding.yaml
+#  sed_wrap -i -e 's/\(: istiod-service-account\)/\1-{{ .Values.revision | default "default" }}/' \
```

Comment:
it looks like none of these changes will be required anymore - should we just remove them entirely instead of commenting?

Reply:
Done.